src/controllers/userController.js

```diff
@@ -59,5 +61,38 @@ const signUp = async (req, res) => {
   });
 };
 
+/**
+   * user login controller
+   * @param {Object} req - server request
+   * @param {Object} res - server response
+   * @return {Object} - custom response
+   */
+const login = async (req, res) => {
+  const {
+    email, password
+  } = req.body;
+
+  const authErrorMessage = [{ field: 'email or password', message: 'email or password is incorrect' }];
+
+  const foundUser = await User.findOne({ where: { email } });
```

Comment:
I observed a user not verified can log in, I feel it would be better if there is a check for isVerified in the User model also.